docs/android/get-started/installation/openjdk.md

```diff
@@ -61,16 +60,18 @@ Point the IDE to the new JDK:
 
 ![Setting the JDK path for the Microsoft Mobile OpenJDK distribution on Windows](openjdk-images/vs.png)
 
-## Revert
+## Known Issues
+
+This is a new feature that affects all Android developers, and we are working hard to resolve issues as quickly as possible. Here are some of the known scenarios and how you can work through them to unblock yourself while we fix things on our end.
```

Comment:
avoid "we" if possible, and "unblock" (for localization). This para could probably be much shorter - the "Known Issues" heading is not uncommon.